File: flink-elasticsearch-connector/src/main/java/com/alex/space/connector/RequestIndexer.java
package com.alex.space.connector;

import java.io.Serializable;
import org.elasticsearch.action.ActionRequest;

/**
 * @author Alex Created by Alex on 2018/9/14.
 */
public interface RequestIndexer extends Serializable {

  void add(ActionRequest... actionRequests);
  
}

File: hbase-to-elastic-flink/src/main/scala/com/alex/space/flink/elasticsearch/RequestIndexer.scala
package com.alex.space.flink.elasticsearch

import java.io.Serializable

import org.elasticsearch.action.ActionRequest

/**
  * @author Alex Created by Alex on 2018/9/14.
  */
trait RequestIndexer extends Serializable {
  def add(actionRequests: ActionRequest*): Unit
}
